error.go

```diff
@@ -294,6 +294,23 @@ func (e *RateLimitError) Error() string {
 	return e.stripeErr.Error()
 }
 
+// redact returns a copy of the error object with sensitive fields replaced with
+// a placeholder value
+func (e *Error) redact() *Error {
```

Comment:
Not a big deal, but could consider a fast path here like:

``` go
if e.PaymentIntent == nil && e.SetupIntent == nil {
    return e
}
```

(Since that's probably like 95%+ of calls to this function.)